packages/gatsby/src/schema/infer-graphql-type.js

```diff
@@ -129,7 +129,7 @@ function inferGraphQLType({
         },
       },
       resolve(object, { fromNow, difference, formatString }) {
-        const date = object[fieldName]
+        const date = object[fieldName].toString()
```

Comment:
does `toISOString()` make more sense as a default? its a bit less culture dependent

Reply:
Oh for sure!

Reply:
Will fix later

Reply:
Needed to do JSON.parse(JSON.stringify(date)) as date could just be a string so this seemed simplest. The internets say that JSON.stringify uses Date.toISOString under the hood anyways.